k2/python/k2/rnnt_loss.py

```diff
@@ -255,6 +260,24 @@ def rnnt_loss_simple(
         boundary=boundary,
         modified=modified,
     )
+
+    if delay_penalty > 0.0:
+        B, S, T0 = px.shape
+        T = T0 if modified else T0 - 1
+        if boundary is None:
+            offset = torch.tensor(
+                [(T - 1) / 2 * delay_penalty] * B,
```

Comment:
If I were you I would multiply by delay_penalty after combining with the offset, to avoid an extra op.

Reply:
Will fix, thanks!